### ISSUE TITLE
Crash on level data update (M420 V)

### ISSUE THREAD

**Issue by disconn3ct** [jneilliii/OctoPrint-BedLevelVisualizer#159]
When clicking to update the mesh, I get a backtrace (and a timeout in the UI.) It doesn't appear to be sending the commands.

The gcode is configured to only:
```
@BEDLEVELVISUALIZER
M420 V
```
(Wrapping it with `M155 S30` .. `M155 S5` sends those commands but it never sends `M420 V`)

Logs show:
```
2019-09-30 10:04:11,308 - Recv:  T:30.78 /0.00 B:49.70 /0.00 @:0 B@:0

==> logs/octoprint.log <==
2019-09-30 10:04:12,558 - octoprint.util.comm - ERROR - Error while processing hook dashboard for phase queuing and command @BEDLEVELVISUALIZER:
Traceback (most recent call last):
  File "/home/pi/oprint/local/lib/python2.7/site-packages/octoprint/util/comm.py", line 3220, in _process_command_phase
    hook_results = hook(self, phase, command, command_type, gcode, subcode=subcode, tags=tags)
  File "/home/pi/oprint/local/lib/python2.7/site-packages/octoprint_dashboard/__init__.py", line 117, in process_gcode
    CmdDict = dict ((x,float(y)) for d,x,y in (re.split('([A-Z])', i) for i in cmd.upper().split()))
  File "/home/pi/oprint/local/lib/python2.7/site-packages/octoprint_dashboard/__init__.py", line 117, in <genexpr>
    CmdDict = dict ((x,float(y)) for d,x,y in (re.split('([A-Z])', i) for i in cmd.upper().split()))
ValueError: too many values to unpack
2019-09-30 10:04:12,562 - octoprint.util.comm - ERROR - Error while processing hook dashboard for phase queuing and command M420 V:
Traceback (most recent call last):
  File "/home/pi/oprint/local/lib/python2.7/site-packages/octoprint/util/comm.py", line 3220, in _process_command_phase
    hook_results = hook(self, phase, command, command_type, gcode, subcode=subcode, tags=tags)
  File "/home/pi/oprint/local/lib/python2.7/site-packages/octoprint_dashboard/__init__.py", line 117, in process_gcode
    CmdDict = dict ((x,float(y)) for d,x,y in (re.split('([A-Z])', i) for i in cmd.upper().split()))
  File "/home/pi/oprint/local/lib/python2.7/site-packages/octoprint_dashboard/__init__.py", line 117, in <genexpr>
    CmdDict = dict ((x,float(y)) for d,x,y in (re.split('([A-Z])', i) for i in cmd.upper().split()))
ValueError: could not convert string to float:

==> logs/serial.log <==
2019-09-30 10:04:13,308 - Recv:  T:30.76 /0.00 B:49.61 /0.00 @:0 B@:0
```

**Comment by jneilliii** [comment]
This error is due to the dashboard plugin, not my plugin.  @StefanCohen, here's an error for you that's part of your gcode hook related to your other issue I think. @disconn3ct, are you using dashboard version 1.7.0, I think some of the gcode parsing issues were fixed with that version.

**Comment by disconn3ct** [comment]
Looks like this is unrelated, a dashboard plugin update came in seconds after I posted that addresses it.

Wow, fast response! Sorry for the misdirected report. (It is still true that as soon as you open a bug report somewhere, you get a solution somewhere else 😹)

**Comment by StefanCohen** [comment]
Um. Yes. Should be fixed in Dashboard v1.6.1 and later.
//Stefan
